synapse/config/room.py

```diff
@@ -50,7 +50,12 @@ def read_config(self, config, **kwargs):
                 RoomCreationPreset.PRIVATE_CHAT,
                 RoomCreationPreset.TRUSTED_PRIVATE_CHAT,
             ]
-        elif encryption_for_room_type == RoomDefaultEncryptionTypes.OFF:
+        elif (
+            encryption_for_room_type == RoomDefaultEncryptionTypes.OFF
+            or encryption_for_room_type is False
+        ):
+            # PyYAML translates "off" into False if it's unquoted, so we also need to
+            # check for encryption_for_room_type being False.
```

Comment:
This is not a PyYAML quirk; [this is part of YAML itself](https://yaml.org/type/bool.html); [c.f. the famous Norway Problem.](https://hitchdev.com/strictyaml/why/implicit-typing-removed/)